staging/src/k8s.io/legacy-cloud-providers/vsphere/vsphere_test.go

```diff
@@ -206,6 +206,11 @@ func configFromEnvOrSim() (VSphereConfig, func()) {
 	return configFromSim()
 }
 
+func init() {
+	klog.Flush()
```

Comment:
You can remove `klog.Flush()`. There's nothing to flush yet.

Reply:
Done